bundles/CoreBundle/Migrations/Version20211103055110.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
```

Comment:
Please add the pimcore copyright and license info like here: https://github.com/pimcore/pimcore/blob/f1488674f12bb9f79f157c167337bff4ae2240fd/bundles/CoreBundle/Migrations/Version20201014101437.php#L3-L14